vscode/install.bash

```diff
@@ -1,8 +1,30 @@
 #! /usr/bin/env bash
 
-if [ ! -f /etc/apt/sources.list.d/vscode.list ]
+# Install vscode via snap
+if [[ ! $(snap list) == *code* ]]
 then
-	wget "https://go.microsoft.com/fwlink/?LinkID=760868" -O /tmp/vscode.deb
-	sudo dpkg -i /tmp/vscode.deb
-	sudo apt-get install --assume-yes -f # Install dependencies
+    tue-install-snap-now code
```

Comment:
We don't have `tue-install-snap-now`

Reply:
Ow, Let me backport that first then. No idea why I didn't catch that in the diff.